packages/app/cypress/e2e/settings.cy.ts

```diff
@@ -147,7 +147,12 @@ describe('App: Settings without cloud', () => {
     cy.findByText('Project Settings').click()
 
     cy.get('[data-cy=config-code]').within(() => {
-      cy.contains('browsers: chrome, firefox')
+      const { browsers } = Cypress.config()
+
+      expect(browsers).to.have.length.greaterThan(1)
+
+      // TODO: is this really how this is supposed to look?
```

Comment:
Opened a Jira ticket for this: https://cypress-io.atlassian.net/browse/UNIFY-1026

```suggestion
```